libexec/erb

```diff
@@ -171,6 +171,4 @@ EOU
   end
 end
 
-if __FILE__ == $0
```

Comment:
Why did you remove this?

Reply:
The file was previously located in `bin/erb`, so that `__FILE__` and `$0` did match. Since https://github.com/ruby/ruby/commit/8c97883b738ad9749848d9a10fce87df0f9b1bf3 `bin/erb` is a binstub which loads `libexec/erb` from the gem directory, so that `__FILE__` and `$0` are different. Running `erb` just requires the code otherwise but doesn't execute it.

Reply:
Thanks, that sounds legit. I'll wait for @hsbt's reaction though, because he made the libexec change.

Reply:
@larskanis Can you separate this change to the another pull-request from this?

Reply:
or rebase from the current master branch.

Reply:
@hsbt Thanks for the fix! Rebased.